### ISSUE TITLE
Allow for custom text on banner

### ISSUE THREAD

**Issue by sjennison** [Maddoc42/Android-Material-Icon-Generator#11]
It would be nice to allow for custom text on the banner

**Comment by Maddoc42** [comment]
Agreed :+1:

Any thoughts on what should happen when the text becomes longer than the banner / icon? Resize the text? Or simply ignore since nobody will be able to read it anyways :smile:

**Comment by sjennison** [comment]
Yeah, I feel like ignoring it would be best (maybe set a max size on the input?).

**Comment by Maddoc42** [comment]
Maxe size on the input is neat, nice!

Anyone coming across this, PRs are welcome :)

**Comment by iskandergaba** [comment]
Whether you are going to implement it using max size or not, I feel like adding a "FREE" and "PRO" default presets makes sense

**Comment by Maddoc42** [comment]
Fair enough :+1: "free" and "pro" are up and running :smile: 

![icon-free-re](https://cloud.githubusercontent.com/assets/1794729/23335022/16db481c-fbac-11e6-942c-5b4bd7227807.png) ![icon-pro-re](https://cloud.githubusercontent.com/assets/1794729/23335024/18f8d7c2-fbac-11e6-8161-05d7394c20b3.png)
